BEPs/BEP-410.md

```diff
@@ -0,0 +1,104 @@
+<pre>
+  BEP: 410
+  Title: Add Agent for Validators
+  Status: Draft
+  Type: Standards
+  Created: 2024-07-15
+</pre>
+
+# BEP-410: Add Agent for Validators
+
+- [BEP-410: Add Agent for Validators](https://github.com/bnb-chain/BEPs/pull/410)
+    - [1. Summary](#1-summary)
+    - [2. Abstract](#2-abstract)
+    - [3. Status](#3-status)
+    - [4. Motivation](#4-motivation)
+    - [5. Specification](#5-specification)
+    - [6. License](#6-license)
+
+
+## 1. Summary
+
+This BEP attempts to enhance the security of validator account management through account isolation by delegating some of the validator operator's permissions to other accounts.
+
+## 2. Abstract
+
+Currently, validators need to manage the operator key, consensus key, and vote key. Among these, the operator key is used for operating a validator, including creating a validator, editing the information of a validator, and undelegating. 
+When creating a validator, the operator key is also used for self-delegating with more than 2001 BNB. Since the operator key holds a large amount of staking, it is not suitable for daily management of the validator. 
+Therefore, we have introduced a new `updateAgent` interface in the system contract. Through this interface, the operator can delegate another account to manage the validator.
+
+## 3. Status
+
+This BEP is in progress.
+
+## 4. Motivation
+
+Before this BEP, the consensus address, commission rate, validator description and vote address can only be changed by the operator which also manages staking for the validator.
+To reduce the usage of operator key and improve the security of operator key management, validators can set an agent to help them to take on the mentioned tasks below.
+
+## 5. Specification
+### Data Structure
+The current `Validator` struct is as follows.
```

Comment:
Can we give a link to the source code of the system contract?

Reply:
added